cfme/containers/image.py

```diff
@@ -39,6 +42,22 @@ def get_detail(self, *ident):
         navigate_to(self, 'Details')
         return InfoBlock.text(*ident)
 
+    def perform_smartstate_analysis(self, wait_for_finish=False):
+        """Performing SmartState Analysis on this Image
+        """
+        navigate_to(self, 'Details')
```

Comment:
I've added """paginator.results_per_page(1000)""" to the step function of the navmazing class. hope this is good enough.